commands-yml/commands/session/create.yml

```diff
@@ -16,9 +16,9 @@ example_usage:
         desiredCapabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "iPhone Simulator");
         desiredCapabilities.setCapability(MobileCapabilityType.AUTOMATION_NAME, "XCUITest");
         desiredCapabilities.setCapability(MobileCapabilityType.APP, "/path/to/ios/app.zip");
-
-        URL url = new URL("http://127.0.0.1:4723");
-
+       
```

Comment:
blank spaces in L19, 20 and 21

Reply:
L19 and 21 already had spaces.
as in http://appium.io/docs/en/commands/session/create/
In my opinion it should be left as it is since it increase the readability

Reply:
I meant below highlighted diff.
![image](https://user-images.githubusercontent.com/5511591/74798909-68bdb000-5312-11ea-89ef-45b8d3d26c2a.png)

I expected the diff was only `/wd/hub`